test/Test/Time/Property.hs

```diff
@@ -88,17 +87,15 @@ rationalNum = do
                              else natural 1
     return $ numeratorVal % denomVal
 
-numRatPair :: (Monad m) => PropertyT m (Int, RatioNat)
-numRatPair = liftA2 (,) (forAll number0'9) (forAll rationalNum)
```

Comment:
You could write this as `liftA2 unitChooser (forAll number0'9) (forAll rationalNum)` if you like it more. But current implementation is good to me. And it's quite readable which is good.